arkouda/dataframe.py

```diff
@@ -1246,35 +1246,19 @@ def save(self, path, index=False):
             File path to save data
         index : bool
             If True, save the index column. By default, do not save the index.
-
-        Notes
-        -----
-        This method saves one file per locale of the arkouda server. All
-        files are prefixed by the path argument and suffixed by their
-        locale number.
-        """
-        tosave = {k: v for k, v in self.data.items() if (index or k != "index")}
-        save_all(tosave, path)
-
-    def save_table(self, prefix_path, columns=None, index=False, file_format='HDF5'):
-        """
-        Save a dataframe as a table in Parquet
-
-        Parameters
-        __________
-        prefix_path: str
-            Path and filename prefix to save to
         columns: List
             List of columns to include in the file. If None, writes out all columns
         file_format: str
             'HDF5' or 'Parquet'. Defaults to 'HDF5'
-        index: Bool
-            If true, include the index values in the save file.
 
         Notes
-        ______
-        This function currently uses 'truncate' mode to ensure the file exists before appending.
+        -----
+        This method saves one file per locale of the arkouda server. All
+        files are prefixed by the path argument and suffixed by their
+        locale number.
         """
+        # tosave = {k: v for k, v in self.data.items() if (index or k != "index")}
+        # save_all(tosave, path)
```

Comment:
are these commented out lines left in intentionally?

Reply:
Nope - these were supposed to to be removed. I left them initially in case moving the code from `save_table` to `save` caused problems. I removed them. Everything should be good now.